library/src/scala/util/FromDigits.scala

```diff
@@ -158,7 +158,7 @@ object FromDigits {
     def fromDigits(digits: String, radix: Int): BigInt = BigInt(digits, radix)
   }
 
-  given BigDecimalFromDigits : FromDigits.Decimal[BigDecimal] {
+  given BigDecimalFromDigits : FromDigits.Floating[BigDecimal] {
```

Comment:
This looks like the correct fix, but then in which case would you actually use `FromDigits.Decimal` ? It seems like it's not really useful on its own /cc @odersky

Reply:
I guess if there is no sparse representation for the implementation then it could make sense.